inputfiles/knownTypes.json

```diff
@@ -0,0 +1,111 @@
+{
+    "Window": [
+        "AAGUID",
```

Comment:
not sure i understand why we need these

Reply:
I'm not sure too, it's just migrated from the old F#-age list. Let's just remove them.